ui/raidboss/data/03-hw/alliance/weeping_city.js

```diff
@@ -0,0 +1,312 @@
+      // Probably kills the player if failed, so it gets an alert.
+      alertText: {
+        en: 'Stand on webs',
+      },
+    },
+    {
+      id: 'Weeping City Punishing Ray',
+      regex: /Punishing Ray/,
+      beforeSeconds: 10,
+      infoText: {
+        en: 'Get Puddles',
+        de: 'Flächen nehmen',
+        fr: 'Allez dans les zones au sol',
+        ja: '踏む',
+        cn: '踩圈',
+        ko: '바닥 징 밟기',
+      },
+    },
+    {
+      id: 'Weeping City Bloodied Nail',
+      regex: /Bloodied Nail/,
+      beforeSeconds: 4,
+      condition: Conditions.caresAboutPhysical(),
+      suppressSeconds: 10,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Weeping City Split End',
+      regex: /Split End/,
+      beforeSeconds: 4,
+      suppressSeconds: 10,
+      response: Responses.tankCleave(),
+    },
+    {
+      id: 'Weeping City Aura Burst',
+      regex: /Aura Burst/,
+      beforeSeconds: 4,
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+  ],
+  triggers: [
+    {
+      // 2 of the 4 encounters in Weeping City use the 0017 head marker.
+      // 2 of the 4 use the 003E head marker.
+      // Because of this, we restrict those triggers for each boss to activate
+      // only when that boss is in progress.
+      id: 'Weeping City HeadMarker Arachne',
+      netRegex: NetRegexes.message({ line: '.*Queen\'s Room will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.arachneStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City HeadMarker Ozma',
+      netRegex: NetRegexes.message({ line: '.*Gloriole will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.arachneStarted = false;
+        data.ozmaStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City HeadMarker Calofisteri',
+      netRegex: NetRegexes.message({ line: '.*Tomb Of The Nullstone will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.ozmaStarted = false;
+        data.calStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City Sticky Wicket',
+      netRegex: NetRegexes.headMarker({ id: '003C', capture: false }),
+      suppressSeconds: 10,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Weeping City Shadow Burst',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      condition: function(data) {
+        return data.arachneStarted;
+      },
+      response: Responses.stackOn(),
+    },
+    {
+      id: 'Weeping City Frond Affeared',
+      netRegex: NetRegexes.startsUsing({ id: '183A', source: 'Arachne Eve', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Weeping City Arachne Web',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: function(data, matches) {
+        return data.arachneStarted && data.me == matches.target;
+      },
+      infoText: {
+        en: 'Drop Web Outside',
```

Comment:
Do these need to be outside? I thought they were fine anywhere?

Reply:
They are, sort of, but there are two points against it. One, they inflict a bleed + heavy on anyone standing on them, so it's best to have them placed not-in-groups. Two, in the second half of the encounter, they are used to avoid being sucked in for Widow's Kiss, and the damage from that is proximity-based. Farther out means people are more likely to survive.

That all being said, if you think "web puddle on you" might be better, I'm okay with that. Less prescriptive is probably a good thing.

Reply:
"Drop Web Outside" sounds good, given that!